interface/src/app/sample/sampleMocks.spec.js

```diff
@@ -0,0 +1,97 @@
+angular.module('adage.mocks.sample', [])
+
+.factory('SampleMocks', [function() {
```

Comment:
By the way, since this service just returns an object literal, you can write like this:
```
.value('sampleMocks', { ... });
```
or:
```
.constant('sampleMocks', { ... });
```